cmd/rotate_certs.go

```diff
@@ -74,8 +74,11 @@ func newRotateCertsCmd() *cobra.Command {
 	f.StringVarP(&rcc.resourceGroupName, "resource-group", "g", "", "the resource group where the cluster is deployed (required)")
 	f.StringVarP(&rcc.apiModelPath, "api-model", "m", "", "path to the generated apimodel.json file (required)")
 	f.StringVarP(&rcc.sshFilepath, "ssh", "", "", "the filepath of a valid private ssh key to access the cluster's nodes (required)")
-	f.StringVar(&rcc.masterFQDN, "master-FQDN", "", "FQDN for the master load balancer (required)")
```

Comment:
the master-FQDN flag still needs to be added I believe, even if it's deprecated. In scale.go you left both flags.